src/pybind/mgr/dashboard/frontend/src/app/core/layouts/login-layout/login-layout.component.html

```diff
@@ -9,14 +9,14 @@
   </header>
   <section>
     <div class="container">
-      <div class="row full-height vertical-align">
-        <div class="col-sm-12 col-md-6 d-sm-block">
+      <div class="row full-height">
+        <div class="col-sm-12 col-md-6 d-sm-block login-form">
           <router-outlet></router-outlet>
         </div>
-        <div class="col-sm-12 col-md-6 d-sm-block">
+        <div class="col-sm-12 col-md-6 d-sm-block branding-info">
           <img src="assets/Ceph_Ceph_Logo_with_text_white.svg"
                alt="Ceph"
-               class="img-fluid">
+               class="img-fluid pb-3">
```

Comment:
>Is there any way @Sarthak0702 to only displace these elements when the banner text is defined (or not displace anything and just leave the banner text uncentered)?

@epuertat If we want that then classes like `vertical-align`, `login-form` and `branding-info` will need to be added conditionally based on if `bannerText` from custom-login-component exists of not. Unless we really want it to be centered and are not changing code just to pass test I guess its should be possible (but a bit tricky/hacky).

Reply:
Ok, not worthy. Let's get this merged. Probably we should have explored absolute positioning or a different approach, but definitely not worthy :P